<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Kirans-MacBook-Air.local" name="apiTests.PostInsertMarketPlaceOrder" tests="1" failures="0" timestamp="12 Oct 2016 11:16:19 GMT" time="0.318" errors="1">
  <testcase name="InsertMarketPlaceOrderTest" time="0.318" classname="apiTests.PostInsertMarketPlaceOrder">
    <error type="junit.framework.AssertionFailedError" message="
Expected: null
  Actual: PostOrderResponse&lt;message=&quot;order %d is submitted!&quot;, success=true&gt;

--- Found following differences ---
expected: null, actual: PostOrderResponse&lt;message=&quot;order %d is submitted!&quot;, success=true&gt;
">
      <![CDATA[junit.framework.AssertionFailedError: 
Expected: null
  Actual: PostOrderResponse<message="order %d is submitted!", success=true>

--- Found following differences ---
expected: null, actual: PostOrderResponse<message="order %d is submitted!", success=true>

	at junit.framework.Assert.fail(Assert.java:50)
	at org.unitils.reflectionassert.ReflectionAssert.assertReflectionEquals(ReflectionAssert.java:136)
	at org.unitils.reflectionassert.ReflectionAssert.assertReflectionEquals(ReflectionAssert.java:99)
	at apiTests.PostInsertMarketPlaceOrder.InsertMarketPlaceOrderTest(PostInsertMarketPlaceOrder.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </error>
  </testcase> <!-- InsertMarketPlaceOrderTest -->
</testsuite> <!-- apiTests.PostInsertMarketPlaceOrder -->
